README.md

```diff
@@ -154,7 +154,7 @@ Notes:
 
 ### Worker join help
 ```
-usage: primazactl worker join [-h] [-x] [-f CONFIG] [-v VERSION] [-c CLUSTER_NAME] [-k KUBECONFIG] -d CLUSTER_ENVIRONMENT -e ENVIRONMENT [-l MAIN_KUBECONFIG] [-m MAIN_CLUSTERNAME]
+usage: primazactl worker join [-h] [-x] [-f CONFIG] [-v VERSION] [-c CONTEXT] [-k KUBECONFIG] -d CLUSTER_ENVIRONMENT -e ENVIRONMENT [-l MAIN_KUBECONFIG] [-m TENANT_CONTEXT]
```

Comment:
Still not sure about Tenant meaning Control Plane. Maybe it will grow on me, so not big deal, but could CONTROL_PLANE_CONTEXT or PRIMAZA_CONTEXT be more obvious than TENANT_CONTEXT. I also saw this in the naming convention PR.